### PR TITLE
Some clean up

### PR DESCRIPTION
**Pull request by mbrandonw**
Fixes the linux crash we are seeing (temporary fix), and cleans up a few other things.

### CODE REVIEW

File: Sources/PointFree/Account/Account.swift [hunk 1]
```diff
@@ -56,12 +56,9 @@ private func fetchAccountData<I, A>(
       .map { $0.right ?? [] },
 
     AppEnvironment.current.database.fetchEmailSettingsForUserId(user.id).run.parallel
-      .map { $0.right ?? [] },
-
-    AppEnvironment.current.database.fetchEpisodeCredits(user.id).run.parallel
```
Comment (mbrandonw) [inline Sources/PointFree/Account/Account.swift]: very strange that a `zip5` crashes on linux. my work to figure out this crash is here https://github.com/pointfreeco/pointfreeco/compare/zip-crash

File: Sources/PointFree/Emails/FreeEpisodeEmail.swift [hunk 2]
```diff
@@ -14,7 +14,7 @@ let freeEpisodeEmail = simpleEmailLayout(freeEpisodeEmailContent)
     SimpleEmailLayoutData(
       user: user,
       newsletter: .newEpisode,
-      title: "Free Point-Free Episode: \(ep.title)",
+      title: "Point-Freebie: \(ep.title)",
```
Comment (mbrandonw) [inline Sources/PointFree/Emails/FreeEpisodeEmail.swift]: I kinda like this play on words. whatdya think?

File: pointfreeco.playground/Pages/About.xcplaygroundpage/Contents.swift [hunk 3]
```diff
@@ -10,12 +10,11 @@ import PlaygroundSupport
 import Prelude
 import WebKit
 import Optics
-import SnapshotTesting
 import Styleguide
 
 AppEnvironment.push(const(.mock))
 
-let request = PointFreeTestSupport.request(to: .pricing(nil, nil))
+let request = PointFreeTestSupport.request(to: .pricing(nil, expand: nil))
```
Comment (mbrandonw) [inline pointfreeco.playground/Pages/About.xcplaygroundpage/Contents.swift]: just fixing a playground

File: pointfreeco.playground/Pages/Episodes Page.xcplaygroundpage/Contents.swift [hunk 4]
```diff
@@ -1,29 +0,0 @@
-import Css
```
Comment (mbrandonw) [inline pointfreeco.playground/Pages/Episodes Page.xcplaygroundpage/Contents.swift]: and deleting old playgrounds that are out of date

File: pointfreeco.playground/Pages/Account.xcplaygroundpage/Contents.swift [hunk 5]
```diff
@@ -14,7 +14,7 @@ import SnapshotTesting
 
 AppEnvironment.push(const(.mock))
 
-let htmlString = render(accountView.view((.mock, .mock, [.mock, .mock], [.mock, .mock, .mock], unit)), config: pretty)
+let htmlString = render(accountView.view((.mock, [.mock, .mock], [.mock, .mock, .mock], [.mock], [.mock], .mock)), config: pretty)
```
Comment (mbrandonw) [inline pointfreeco.playground/Pages/Account.xcplaygroundpage/Contents.swift]: would be nice to get playgrounds building in CI.

File: Sources/PointFree/Emails/FreeEpisodeEmail.swift (hunk 2 above)
Comment (stephencelis) [inline Sources/PointFree/Emails/FreeEpisodeEmail.swift]: "Point-Free Freebie" might read a bit easier.

**Comment by stephencelis** [comment]
(Also fix tests?)

**Pull request merged**